tools/pipelines/build-build-tools.yml

```diff
@@ -59,10 +61,35 @@ extends:
   parameters:
     publishOverride: ${{ parameters.publishOverride }}
     releaseBuildOverride: ${{ parameters.releaseBuildOverride }}
-    buildDirectory: tools/build-tools
+    buildDirectory: build-tools
```

Comment:
@sonalivdeshpande mentioned this as well. The `build-tools` path is correct. I want to build the monorepo/release group in that folder -- which now has a packages/build-tools folder within it. I'd prefer to rename the build-tools package to be something like build-lib, but I can't do that right now because we publish the package and many folks use it from npm. I don't want to rename it if we still need to publish it.